irods/test/collection_test.py

```diff
@@ -322,6 +324,33 @@ def test_register_collection_with_checksums(self):
         shutil.rmtree(dir_path)
 
 
+    def test_collection_with_trailing_slash__323(self):
+        Home = helpers.home_collection(self.sess)
+        subcoll, dataobj = [unique_name(my_function_name(),time.time()) for x in range(2)]
+        subcoll_fullpath = "{}/{}".format(Home,subcoll)
+        subcoll_unnormalized = subcoll_fullpath + "/"
+        try:
+            # issue 323  - test create and exists with trailing slashes
+            self.sess.collections.create(subcoll_unnormalized)
+            c1 = self.sess.collections.get( subcoll_unnormalized )
+            c2 = self.sess.collections.get( subcoll_fullpath )
+            self.assertEqual(c1.id, c2.id)
+            self.assertTrue( self.sess.collections.exists(subcoll_unnormalized))
```

Comment:
inconsistent spacing around function arguments.

there are other places that need to be cleaned up as well.

Reply:
Think these are fine now.